### ISSUE TITLE
replace GitLab base URL in formatted responses with configurable value

### ISSUE THREAD

**Issue by nazrhyn** [nazrhyn/gitlab-slack#8]
I noticed this because of spodlecki/gitlab-slack@a8811a5d. Not sure why I missed this when I was on a crusade against these hard-coded URLs.

Thanks @spodlecki :smile:.

**Comment by spodlecki** [comment]
:+1: I was going to submit a PR, but the new GitLabs has a (somewhat) working version for Slack integrations. Didn't really figure this would be supported much anymore

**Comment by nazrhyn** [comment]
No worries! I thought I wouldn't be maintaining it either, when I saw that they fixed the issues with the built-in support. I turned it on and tried it for our team, but ultimately didn't like the way that the notifications were formatted as much as I liked my formatting, though.

Additionally, #3 and #1 are features that are unique to my implementation that my team likes. Beyond that, I'm working on #7 which is going to be huge for us (as GitLab's webhook information just isn't enough to do diffs properly).

I should probably put a features list in the README, eh?
